src/main/java/com/crud/library/service/BorrowedBookService.java

```diff
@@ -0,0 +1,33 @@
+package com.crud.library.service;
+
+import com.crud.library.domain.BorrowedBook;
+import com.crud.library.repository.BorrowedBookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@Transactional
```

Comment:
tu `@Transactional` nie ma sensu, bo metody w implementacji repozytorium same w sobie są transakcyjne. jeśli robiłbyś w tych metodach coś więcej to wtedy tak.